### PR TITLE
TF-555 - Add swipes and filters

### PR DESCRIPTION
**Pull request by GabrielHomewoodUL**
Add swipes and filters

### CODE REVIEW

File: TF_Service_dotNet/TouchFree/Interactions/InteractionModules/VelocitySwipeInteraction.cs [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+            }
+
+            return false;
+        }
+
+        protected override void OnInteractionSettingsUpdated(InteractionConfigInternal _config)
+        {
+            base.OnInteractionSettingsUpdated(_config);
+
+            //minScrollVelocity_mmps = _config.minScrollVelocity_mmps;
```
Comment (NatGreenwood-UL) [inline TF_Service_dotNet/TouchFree/Interactions/InteractionModules/VelocitySwipeInteraction.cs]: I _think_ these are all in the InteractionConfig file atm so do they need to be commented out?

**Comment by NatGreenwood-UL** [comment]
PR looks good, going to try out the interactions locally and will throw any further feedback on here

File: TF_Service_dotNet/TouchFree/Interactions/InteractionModules/VelocitySwipeInteraction.cs (hunk 1 above)
Comment (GabrielHomewoodUL) [reply TF_Service_dotNet/TouchFree/Interactions/InteractionModules/VelocitySwipeInteraction.cs]: These were added before I created the interaction tuning json configuration file. I'll remove these as we will probably want a different way of configuring this interaction when we add the settings into the Settings UI

Comment (GabrielHomewoodUL) [reply TF_Service_dotNet/TouchFree/Interactions/InteractionModules/VelocitySwipeInteraction.cs]: I have removed the references to these settings from the interaction configuration file now

**Review by NatGreenwood-UL — APPROVED** [review]

**Pull request merged**